server.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const EXPRESS = require('express');
+const PORT = 3000;
```

Comment:
You should include `process.env.PORT` here in order for this to work when the server is hosted elsewhere.